params/protocol_params.go

```diff
@@ -160,6 +164,9 @@ const (
 	RefundQuotient        uint64 = 2
 	RefundQuotientEIP3529 uint64 = 5
 
+	HistoricalRootsModulus  uint64 = 98304 // Limits how many historical roots are stored by EIP-4788
```

Comment:
I'd recommend starting to prefix constants belonging to the same EIP/fork/whatnot since we already have tons of params and it's very hard to see what goes where. I.e. BeaconRootHistoricalRootsModulus